apps/ui/src/components/search-box.tsx

```diff
@@ -0,0 +1,19 @@
+import { Search } from "../icons/search";
+
+interface SearchBoxProps {
+  handleSearch: (value: string) => void;
+}
+
+export function SearchBox({ handleSearch }: SearchBoxProps) {
+  return (
+    <div className="flex items-center gap-2 border border-accent  py-1 px-2 rounded-md">
+      <Search width={15} height={15} />
+      <input
+        type="text"
+        className="bg-transparent focus:outline-none"
+        placeholder="Search"
+        onChange={({ target: { value } }) => handleSearch(value)}
+      />
+    </div>
```

Comment:
Can we add an `X` type icon to clear the search field? Not sure what component library we use but this is probably a built-in feature.